app/agents/validator.py

```
@@ -0,0 +1,272 @@
+            state["should_continue"] = not validation_result["is_valid"]
+
+        except Exception as e:
+            logger.error(f"Error in validation: {e}")
+            state["validation_results"] = {
+                "is_valid": False,
+                "error": f"Error en validación: {str(e)}"
+            }
+
+        return state
+
+    def _validate_by_type(self, validation_type: str, value: str) -> Dict[str, Any]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)